### PR TITLE
Core 1290 new matched loans position relending test

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- matched loans section moved up to quick filters for relending test
- copy updated matched loans section for test
- data prefetched avoiding async behavior in this section

### CODE REVIEW

File: src/pages/LoanFinding/LoanFinding.vue [hunk 1]
```diff
@@ -147,9 +156,27 @@ export default {
 					variables: prefetchedRecommendedLoansVariables
 				});
 
+				const matchedLoansPromise = client.query({
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:159]: Why are we prefetching the second row? Is it ever visible above the fold when the page first loads?

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:159]: I think it depends of screen resolution, its ever visible for the test for myself

Screenshot 2023-05-04 at 14 19 47

Comment (dyersituations) [reply src/pages/LoanFinding/LoanFinding.vue:159]: How much do the extra prefetch queries affect the Lighthouse scores locally?

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:159]: Nice call! Ill test it in a while...

**Review by dyersituations — APPROVED** [review]

**Pull request merged**